config/environments/development.rb

```diff
@@ -1,9 +1,10 @@
-# rubocop:disable Lint/MissingCopEnableDirective, Metrics/BlockLength
-Rails.application.configure do
+require 'active_support/core_ext/integer/time'
+
+Rails.application.configure do # rubocop:disable Metrics/BlockLength
```

Comment:
[Nit] Any particular reason for preferring to do the override inline? Most of the others in the app are above. Obviously not a blocker, mostly curious :sweat_smile: 

Worth either adding a small `.rubocop.yml` to the config dir to ignore it, or adding an exclude rule to the main linting config?

Reply:
Just a change of preference lately, mostly down to laziness and avoiding needing to have two lines 😆 

Adding a rule to exclude the environment files is much better, updated. Thanks @ChargrilledChook 💪